Community/Tazama-Contribution-Guide.md

```diff
@@ -486,6 +484,8 @@ Best practices for making changes.
 
 **Code Documentation** – Electronic documentation has been auto-generated (where available) and has been checked for correctness. Also have sufficient code comments. [Developer] + [PR Approver]
 
+**Licensing comment** Add the following string as a comment at the top of every file in the FRMS organisation in GitHub that is capable of including a comment  " SPDX-License-Identifier: Apache-2.0"

```

Comment:
I'd specify the different files and comment formats explicitly here. Technically, JSON files don't typically get commented but we've added the header to ours, so we'll need to tell people explicitly to make sure they add it there too.

Reply:
Can you help with the different file types and comment formats?

Reply:
File types for License header added to contribution guide